src/streaming/protection/models/ProtectionModel_21Jan2015.js

```diff
@@ -178,9 +178,10 @@ function ProtectionModel_21Jan2015(config) {
 
         let session = mediaKeys.createSession(sessionType);
         let sessionToken = createSessionToken(session, initData, sessionType);
+        let ks = this.getKeySystem();
 
         // Generate initial key request
-        session.generateRequest('cenc', initData).then(function () {
+        session.generateRequest(ks.systemString === 'org.w3.clearkey' ? 'keyids' : 'cenc', initData).then(function () {
```

Comment:
I think makes sense declaring a new Constants file, focused on EME stuff, where we declare system strings of the different keySystems so we have just one place to define them all. I am afraid of having these strings (ex: "org.w3.clearkey") in different places.